spec/requests/contest_relations_request_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
+
+RSpec.describe ContestRelationsController, type: :request do
+  let(:student) { FactoryBot.create(:user) }
+  let(:admin) { FactoryBot.create(:superadmin) }
```

Comment:
I think I understand how this works now, but I was a bit confused at first since `admin` isn't always an admin. Might be clearer if renamed to `user` or `current_user`?